### PR TITLE
Init How it Works page

### PR DESCRIPTION
**Pull request by spacesailor24**
Refactors the existing [How Does Lit Protocol Work](https://developer.litprotocol.com/resources/how-it-works) page

### CODE REVIEW

File: docs/learn/overview/how-it-works/overview.md [hunk 1]
```diff
@@ -1 +1,122 @@
...
+- **Threshold Signatures:** Operations like signing and decryption require collaboration among multiple nodes, enhancing security.
+- **Access Control Policies:** Developers can define complex [access control conditions](../../encryption-access-control/overview.md) based on various conditions, enabling fine-grained control over who can access certain data or perform specific actions.
+
+### Confidential Computing with TEEs
+
+The use of TEEs ensures that computations are performed securely and privately:
+
+- **Data Privacy:** Sensitive data processed within a TEE remains confidential, as it cannot be accessed by external parties or the node operator.
+- **Code Integrity:** The code executing within the TEE is protected from tampering, ensuring consistent and trustworthy operations.
+- **Verification:** Clients can verify that nodes are running the correct code within a TEE through remote attestation mechanisms.
```
Comment (spacesailor24) [inline docs/learn/overview/how-it-works/overview.md:111]: Not sure how factual this is

File: docs/learn/overview/how-it-works/overview.md [hunk 2]
```diff
@@ -1 +1,122 @@
...
+
+At the heart of Lit Protocol are the **Lit Nodes**, which are independently operated servers that collectively form the Lit Network. These nodes perform the computational and cryptographic operations necessary for the protocol's functionality.
+
+### Trusted Execution Environments (TEEs)
+
+Each Lit Node runs within a **Trusted Execution Environment (TEE)** provided by [AMD's SEV-SNP](https://www.amd.com/content/dam/amd/en/documents/epyc-business-docs/solution-briefs/amd-secure-encrypted-virtualization-solution-brief.pdf) technology. TEEs offer a secure area within a computer's processor, ensuring that code and data loaded inside are protected. This allows for:
+
+- **Confidentiality:** Data processed inside the TEE is encrypted and cannot be accessed by the node operator or any external entity.
+- **Integrity:** The code running inside the TEE cannot be tampered with, ensuring that nodes execute the correct software.
+- **Remote Attestation:** TEEs provide mechanisms for verifying that the code running within them is genuine and has not been altered.
```
Comment (spacesailor24) [inline docs/learn/overview/how-it-works/overview.md:25]: Not sure how factual this is

File: docs/learn/overview/how-it-works/overview.md [hunk 3]
```diff
@@ -1 +1,122 @@
...
+
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# How Lit Protocol Works
+
+Building upon the foundational concepts introduced in the [What is Lit Protocol](../what-is-lit) page, this section delves deeper into the technical architecture and components that make Lit Protocol function under the hood.
+
+Lit Protocol combines advanced cryptography, confidential hardware, and peer-to-peer networking to provide a decentralized platform for key management and secure computation. Below, we explore the core components and mechanisms that enable these capabilities.
+
+## The Lit Nodes
```
Comment (awisniew207) [inline docs/learn/overview/how-it-works/overview.md:15]: Maybe we add the list of node operators here?:
https://developer.litprotocol.com/resources/how-it-works#node-operators

Comment (awisniew207) [reply docs/learn/overview/how-it-works/overview.md:15]: Nevermind, didn't read far enough

File: docs/learn/overview/how-it-works/overview.md [hunk 4]
```diff
@@ -1 +1,122 @@
...
+### Secure Key Management Without Custodians
+
+Lit Protocol provides a decentralized alternative to traditional key management solutions, which often rely on centralized custodians or require users to manage their own private keys.
+
+- **Non-Custodial Key Generation:** Users can have keys generated and managed securely without any single entity controlling the full private key.
+- **Flexible Authentication Methods:** Lit allows for key generation using traditional Web2 authentication methods like social OAuth (e.g. Google, Facebook, X (Twitter)). This enables seamless user onboarding by leveraging familiar login mechanisms while maintaining decentralized key management.
+- **Threshold Signatures:** Operations like signing and decryption require collaboration among multiple nodes, enhancing security.
+- **Access Control Policies:** Developers can define complex [access control conditions](../../encryption-access-control/overview.md) based on various conditions, enabling fine-grained control over who can access certain data or perform specific actions.
+
+### Confidential Computing with TEEs
```
Comment (awisniew207) [inline docs/learn/overview/how-it-works/overview.md:105]: I feel like this section is very similar to the TEE section earlier in this page

**Pull request merged**